contents/blog/data-management-feature.md

```diff
@@ -0,0 +1,83 @@
+- Categorizing your definitions with tags
+- Giving custom names and descriptions to events and properties
+- Making the right events and properties discoverable with event metrics
+
+## What is an Event?
+
+[Events](/docs/user-guides/events) are the basic building blocks of data that is sent to PostHog. Events can capture a wide variety of different interactions, from a new user signing up to capturing a server event when an API endpoint is called.
+
+[Event Properties](/docs/user-guides/events#event-properties) make up the meat of the event. These can be viewed nested in the context of a single event, or as a flat list in the Data Management > Event Properties tab.
+
+## How to create event and event property definitions
+
+### Creating definitions in the Insight Builder (on the fly!)
+
+Building an insight and see an event or event property that could be more descriptive? You can now create a definition for that without leaving the insight query builder.
+
+![PostHog - read definition hover card](../images/blog/data-management-feature/read_definition_card.png)
+
+![PostHog - write definition hover card](../images/blog/data-management-feature/write_definition_card.png)
+
+### Creating definitions in the new Data Management Tab
+
+Alternatively, click on view in the definition hover card and you'll be redirected to the new "Data Management" page. From here, explore a centralized place to manage all the event and event property definitions you have created.
+
+[WIP screenshot]
+
+### Creating definitions with the API
+
+You can also opt to create definitions with our API. Check out our documentation on how to use the [`event_definitions`](/docs/api/event-definitions) and [`property_definitions`](/docs/api/property-definitions) API endpoints.
+
+## How to use Data Management as a team (Best practices)
+
+1. **Rule of thumb**: The more definitions you define, the easier it will be for your teammates to understand what each event and property is describing. 
+
+Take a custom event sent as `sign_up` for example that is triggered on user sign up. Pretty self explanatory, right? Imagine you have a second app pushing data to the same PH instance sending sign up events as `sign up`. A new teammate using PH will find it impossible to differentiate between the two, leading to incorrect queries and onboarding woes. Defining two separate definitions (with different custom names, descriptions, tags) can be very powerful in these situations, not only in documenting your data, but also in providing clarity to teammates.
+
+2. **Rule of thumb**: Official organization-wide events should be verified so that teammates can discover and use them easily. 
+
+Verified events will be prioritized in filters and other selection components to signal to the rest of the team that this event should be used in favor of other similar events.
+
+3. **Rule of thumb**: Tagging is an easy way to categorize events into custom groups, be it for a product, feature or new page. 
```

Comment:
I would take a very opinionated approach here where we share best practices on what to tag. Example: scope tagging (`product`, `marketing`, `website`, ...)